website/docs/src/content/quickstart.mdx

```diff
@@ -341,10 +341,233 @@ const stream = await workflow.run({ topic: "latest quantum computing chips" });
 
 ![blog writing trace](/quickstart/blog-trace.png)
 
-## Next steps
+## Running the dev server
 
-Now that you've gone through the quickstart, you should be able to start building with GenSX. Take a look at the following examples to see how you can build more complex workflows.
+GenSX provides a local development with local REST APIs that match the shape of workflows deployed to GenSX Cloud. Run the dev server from the CLI:
+
+```bash
+# Start the development server
+gensx start src/workflows.tsx
+```
+
+The development server provides several key features:
+
+- **Hot reloading**: Changes to your code are automatically detected and recompiled
+- **API endpoints**: Each workflow is exposed as a REST endpoint
+- **Swagger UI**: Interactive documentation for your workflows at `http://localhost:1337/swagger-ui`
+- **Local storage**: Built-in support for blob storage and databases
+
+You'll see something like this when you start the server:
+
+```bash
+🔍 Starting GenSX Dev Server...
+ℹ Using project name from gensx.yaml: my-project
+ℹ Starting development server...
+✔ Compilation completed
+✔ Generating schema
+
+📋 Available workflows:
+- ResearchAndWriteWorkflow: http://localhost:1337/local/my-project/default/workflows/ResearchAndWriteWorkflow/run
+
+✅ Server is running. Press Ctrl+C to stop.
+```
+
+You can now test your workflow by sending requests to the provided URL using any HTTP client, or using the built-in Swagger UI at `http://localhost:1337/swagger-ui`.
+
+## Deploying your project to GenSX Cloud
+
+GenSX Cloud provides serverless deployment with zero configuration:
+
+```bash
+# Deploy your project to GenSX Cloud
+gensx deploy src/workflows.tsx -ev OPENAI_API_KEY
+```
+
+This command:
+
+1. Builds your TypeScript code for production
+2. Bundles all dependencies
+3. Uploads the package to GenSX Cloud
+4. Creates REST API endpoints for each workflow
+5. Configures serverless infrastructure
+
+For production deployments, you can target a specific environment:
+
+```bash
+# Deploy to production environment
+gensx deploy src/index.ts --environment production
+```
+
+### Running a workflow from the CLI
```

Comment:
Leading with the CLI feels strange here. I think a better path for this quickstart would be
a) go test it in the console
b) copy the code and go run it

I think we should show sample code for calling a workflow somewhere here because ultimately we want people calling workflows programatically

Reply:
Not even sure we need to mention the CLI here (less is more)